src/nano_contracts/builder.ts

```diff
@@ -34,7 +39,7 @@ class NanoContractTransactionBuilder {
 
   caller: Buffer | null;
 
-  args: any[] | null;
+  args: NanoContractArgumentType[] | null;
```

Comment:
We don't expect `Buffer` here

Reply:
Changed to the new type mentioned above.